### DISCUSSION TITLE
hitsound volume inconsistency

### DISCUSSION THREAD

**Discussion by u3o** [ppy/osu#27311] [Q&A]
sometimes (mostly when i play poorly) some hitsounds are getting too loud. im guessing that it is because a few hitsounds get layered on one another and not sure if its intentional or not 

https://github.com/ppy/osu/assets/64749081/1a04be38-4bef-43eb-ae4c-2209107951b3

**Comment by u3o** [comment]
and here are the logs

[1708595775.network.log](https://github.com/ppy/osu/files/14372754/1708595775.network.log)
[1708595775.performance.log](https://github.com/ppy/osu/files/14372755/1708595775.performance.log)
[1708595775.runtime.log](https://github.com/ppy/osu/files/14372756/1708595775.runtime.log)
[1708595775.updater.log](https://github.com/ppy/osu/files/14372757/1708595775.updater.log)
[1708595775.database.log](https://github.com/ppy/osu/files/14372759/1708595775.database.log)

**Comment by peppy** [comment]
Can you make this happen with the default skin? Does it happen on all beatmaps? How often?

**Reply by u3o** [reply]
it only matters what hitsounds the skin uses, its easily noticed while using skins that use "clicky" hitsounds and only use 1 sound for everything (so there is a big chance of 1 sound being played at the same time). Thats why you cant hear this in argon, but you can in the legacy default skin though. but im most certain that this "bug" (i dont really think its a bug) appears because 2 hitsounds are played at the same time (for example when you hit head late in a very short slider). so i think osu doesnt do something wrong, but there should be a limit to how loud hitsounds can be, so even if 2 sounds play at the same time, those sounds dont multiply and become too loud.

**Reply by peppy** [reply]
This is very likely another occurrence of https://github.com/ppy/osu/issues/26879. It's the only way things could be played twice, I think.
